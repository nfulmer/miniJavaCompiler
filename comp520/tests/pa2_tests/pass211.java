// PA2 method decl pass
class Foo {
    private void x(){}
}
