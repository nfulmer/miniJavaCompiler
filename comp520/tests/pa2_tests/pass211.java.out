======= AST Display =========================
Package
  ClassDeclList [1]
  . ClassDecl
  .   "Foo" classname
  .   FieldDeclList [0]
  .   MethodDeclList [1]
  .   . (private) MethodDecl
  .   .   VOID BaseType
  .   .   "x" methodname
  .   .   ParameterDeclList [0]
  .   .   StmtList [0]
=============================================
